src/homepage/index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html" charset="UTF-8" />
+  <title>Jana-pair-programming // Homework 4</title>
+  <link rel="stylesheet" type="text/css" href="navbar.css" />
+  <link rel="stylesheet" type="text/css" href="firstSection.css" />
+  <link rel="stylesheet" type="text/css" href="secondSection.css" />
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <script src="navbar.js"></script>
+</head>
+
+<body>
+  <div class="navbar">
+    <div class="logo">
+      <img src="images/logo.png" alt="logo">
+    </div>
+    <div class="menu">
+      <ul>
+        <li>
+          <a href="#home">Home</a>
+        </li>
+        <li>
+          <a href="#page1">Page1</a>
+        </li>
+        <li>
+          <a href="#page2">Page2</a>
+        </li>
+        <li class="dropdown">
+          <a href="javascript:void(0)" class="dropbtn">Page3</a>
+          <div class="dropdown-content">
+            <a href="#">Link 2</a>
+            <a href="#">Link 3</a>
+            <a href="#">Link 1</a>
+            <li class="dropdown">
+              <a href="javascript:void(0)" class="dropbtn">Page4</a>
+              <div class="dropdown-content">
+                <a href="#">Link 2</a>
+                <a href="#">Link 3</a>
+                <a href="#">Link 1</a>
+                <li>
+                  <a href="#contact">Contact</a>
+                </li>
+      </ul>
+      </div>
+      </div>
+      <div class="background">
+        <div>
+          <h1>Dropdown menu inside<br><div class="blueHeader">a navigation bar</div></h1>
```

Comment:
Na elementy, které zvýrazňují text je hezčí použít nějaký inlinový element - zde například strong, který to hezky zvýrazní i pro čtečky (probereme semantiku na další session)